llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -1725,6 +1744,7 @@ static void hoistConditionalLoadsStores(
     MaskedLoadStore->copyMetadata(*I);
     I->eraseFromParent();
   }
+  return true;
```

Comment:
I think this function always return true, should we change it to void?

Reply:
Done.